one-ui/ui/src/components/load-data/new-data-load-dialog/new-data-load-dialog.test.tsx

```diff
@@ -1,9 +1,89 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { act, render, fireEvent, waitForElement } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
 import NewDataLoadDialog from './new-data-load-dialog';
+import {BrowserRouter} from "react-router-dom";
+import axiosMock from 'axios'
 
-describe('New/Edit data load component', () => {
-  it('should render correctly', () => {
-    shallow(<NewDataLoadDialog data/>);
+jest.mock('axios');
+
+axiosMock.mockImplementation(jest.fn(() => Promise.resolve({ status: 200, data: {} })));
+axiosMock.post.mockImplementation(jest.fn(() => Promise.resolve({ status: 200, data: {} })));
+axiosMock.get.mockImplementation(jest.fn(() => Promise.resolve({ status: 200, data: {} })));
+
```

Comment:
This looks good @ryanjdew . Can we clear the mocks after a test is done?

Reply:
@bsrikan Do you mean something like below to clear the counts?
```
afterEach(() => {
  jest.clearAllMocks();
});
```

Reply:
Right. So it doesnt bleed into other tests that may use those mocks.

Reply:
@bsrikan Ok. Added the snippet above and rebased.